### PR TITLE
allow sending option append_deeplink_path

### PR DESCRIPTION
**Pull request by derrickstaten**
@jsaleigh @jeanmw per a partner's request. This is a very advanced option so let's leave undocumented for now.

### CODE REVIEW

File: test/6_branch.js [hunk 1]
```diff
@@ -856,6 +857,7 @@ describe('Branch', function() {
 			assert.strictEqual(obj.data, '{"mydata":"bar"}', 'data is sent');
 			assert.deepEqual(obj.tags, [ "tag1", "tag2" ], 'tags is sent');
 			assert.strictEqual(obj.open_app, true, 'open_app is sent');
+			assert.strictEqual(obj.append_deeplink_path, true, 'append_deeplink_path is sent');
```
Comment (jsaleigh) [inline test/6_branch.js]: Should this be append_universal_link?

Comment (jsaleigh) [inline test/6_branch.js]: Or should the other be append_deeplink_path?

Comment (derrickstaten) [inline test/6_branch.js]: shit, too late at night. `append_deeplink_path`

**Comment by jsaleigh** [comment]
:+1:

**Comment by jsaleigh** [comment]
Should probably get a thumb from @jeanmw too since i touched the code

**Comment by derrickstaten** [comment]
:+1: your test fix makes sense. @jeanmw please thumb

**Comment by jeanmw** [comment]
@derrickstaten LGTM :+1:

**Comment by jsaleigh** [comment]
Hmm...nope, my tests didn't work either, they broke mobile tests this time :/

**Pull request merged**